MODULE.bazel

```diff
@@ -147,24 +149,25 @@ maven.install(
         "io.netty:netty-transport-native-unix-common:jar:linux-x86_64:4.1.93.Final",
         "io.netty:netty-transport-native-unix-common:jar:osx-aarch_64:4.1.93.Final",
         "io.netty:netty-transport-native-unix-common:jar:osx-x86_64:4.1.93.Final",
-        "io.netty:netty-transport:4.1.93.Final",
         "io.reactivex.rxjava3:rxjava:3.1.2",
         "it.unimi.dsi:fastutil:7.2.1",
         "javax.activation:javax.activation-api:1.2.0",
         "javax.annotation:javax.annotation-api:1.3.2",
         "javax.inject:javax.inject:1",
-        "net.bytebuddy:byte-buddy-agent:1.14.5",
         "net.bytebuddy:byte-buddy:1.14.5",
+        "net.bytebuddy:byte-buddy-agent:1.14.5",
         "org.apache.commons:commons-compress:1.20",
         "org.apache.commons:commons-pool2:2.8.0",
         "org.apache.tomcat:tomcat-annotations-api:8.0.5",
         "org.apache.velocity:velocity:1.7",
         "org.checkerframework:checker-qual:3.19.0",
+        "org.openjdk.jmh:jmh-core:1.37",
```

Comment:
Why do we need this `openjdk.jmh:jmh-core`?

Reply:
It's for the benchmark I added in the linked commit. It's not used by `//src:bazel`.

Reply:
Oh, I see